### PR TITLE
add OWL Carousel

### PR DESCRIPTION
**Pull request by NovikovViktor**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -7005,7 +7005,19 @@
...
+		},
+        "OWL Carousel": {
+            "cats": [
+                5,
+                7
+            ],
+            "html": "<link [^>]*href=\"[^\"]+owl.carousel(?:\\.min)?\\.css",
+            "script": "owl.carousel.*\\.js",
+            "implies": [
+                "CSS", "jQuery"
```
Comment (ElbertF) [inline src/apps.json]: There is no application called `CSS`, please change it to `"implies": "jQuery",`

Comment (DaAwesomeP) [inline src/apps.json]: This is also not in alphabetical order if that matters.

Comment (ElbertF) [inline src/apps.json]: That's ok, running `wappalyzer build` will fix the order and indentation.

**Comment by NovikovViktor** [comment]
@ElbertF , I change `"implies"` part, so it can be merged...

**Comment by ElbertF** [comment]
Thanks!

**Pull request merged**